ragen/env/alfworld/utils.py

```
@@ -4,7 +4,9 @@
 import re
 from typing import List, Any
 
-def load_config(config_file: str, params: List[str] = []):
+def load_config(config_file: str, params: List[str] = None):
```

Ruff (0.15.5)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)